qemu/tests/virtio_fs_share_data.py

```diff
@@ -455,7 +455,24 @@ def viofs_svc_stop_start(action, cmd, expect_status):
                     session.cmd(cmd_make % fs_dest, io_timeout)
                     status, output = session.cmd_status_output(
                         cmd_pjdfstest % fs_dest, io_timeout)
-                    if status != 0:
+                    failed_test = output.split("-------------------")[1]\
+                                        .split("Files=")[0]
+                    # ignore the specific failed cases from pjdfstest
+                    ignore_cases = ["pjdfstest/tests/rename/21.t",
+                                    "pjdfstest/tests/symlink/03.t"]
+                    cases_in_output = list(re.findall(r"/mnt.*\.t", failed_test))
+                    false_in_list = [False for _ in range(len(cases_in_output))]
+                    cases_in_output = dict(zip(cases_in_output, false_in_list))
+                    for case in cases_in_output.keys():
+                        for ig_case in ignore_cases:
+                            if ig_case in case:
+                                error_context.context("Warn: %s was failed!"
+                                                      % ig_case, test.log.debug)
+                                cases_in_output[case] = True
+
+                    unexpected_fail_case = list(cases_in_output.values()).count(False)
+
+                    if status != 0 and unexpected_fail_case > 0:
```

Comment:
I think we could specify the `generic_blacklist` to ignore some unstable cases.
BTW, for the failed case `pjdfstest/tests/rename/21.t` and `pjdfstest/tests/symlink/03.t`,  
I remember they are passed in the previous version, right? @zhenyzha

Reply:
Yes, the failed cases vary by version. These two fail on the current latest version.

Reply:
I checked the previous results on x86, for rename an symlink, the failed case ID is the same.
But I agree to set a blacklist in config file.

2022-02-08 11:09:32: Test Summary Report
2022-02-08 11:09:32: -------------------
2022-02-08 11:09:32: /mnt/myfs/pjdfstest/tests/chown/00.t          (Wstat: 0 Tests: 1323 Failed: 0)
2022-02-08 11:09:32:   TODO passed:   693, 697, 708-709, 714-715, 729, 733
2022-02-08 11:09:32: [31m/mnt/myfs/pjdfstest/tests/rename/21.t         (Wstat: 0 Tests: 16 Failed: 3)[0m
2022-02-08 11:09:32: [31m  Failed tests:  [0m[31m12-14[0m
2022-02-08 11:09:32: [31m/mnt/myfs/pjdfstest/tests/symlink/03.t        (Wstat: 0 Tests: 6 Failed: 2)[0m
2022-02-08 11:09:32: [31m  Failed tests:  [0m[31m1-2[0m
2022-02-08 11:09:32: Files=235, Tests=8813, 198 wallclock secs ( 1.85 usr  0.57 sys + 26.32 cusr 49.33 csys = 78.07 CPU)
2022-02-08 11:09:32: Result: FAIL

Reply:
> Yes, the failed cases vary by version. These two fail on the current latest version.

Then do we need to define the blacklist according to different versions? e.g. 
Host_RHEL.m8, Host_RHEL.m9.u0:
  blacklist = 'xxx'

Reply:
@nickzhq For rhel8 on x86, pls add pjdfstest/tests/utimensat/09.t to blacklist.

@zhenyzha Could you pls add the blacklist from multi-arch perspective?

Thank you.

Reply:
@xiagao Tested and currently consistent with x86. If there are changes in the future, I will add it myself, thanks for the reminder.